lib/theme/toast_messages/hs_toast_theme.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:hatspace/gen/assets.gen.dart';
+import 'package:hatspace/theme/toast_messages/toast_messages.dart';
+
+class ToastMessageError extends StatelessWidget {
+  final String message;
+  final String title;
+  const ToastMessageError(
+      {super.key, required this.message, required this.title});
+  @override
+  Widget build(BuildContext context) {
+    // TODO: implement build
+    return ToastMessageContainer(
```

Comment:
a widget to return just single widget? reconsider to use your ToastType enum and remove extra widget here

Reply:
Need to discuss